demo/mobile/src/app/app.js

```diff
@@ -3,6 +3,10 @@
 var angular = require('angular');
 window.async = require('async');
 window._ = require('underscore');
+var logger = require('@raincatcher/logger');
+
+// Create INFO logger
+logger.setLogger(new logger.ClientLogger(2));
```

Comment:
I did not changed any of this. Just used previous api that was build.

From https://www.npmjs.com/package/loglevel:
```
As a numeric index from 0 (trace) to 5 (silent) ← deliciously terse, and more easily programmable (...although, why?)
```